Makefile

```diff
@@ -38,6 +38,7 @@ eunit:
 	${REBAR} as test eunit
 
 ct:
+	openssl rand -out test/partisan_SUITE_data/RAND 4096
```

Comment:
`make_certs` requires this file, it's seed entropy for certificate and key generation.